components/MyBookings.js

```diff
@@ -0,0 +1,57 @@
+import { useEffect } from 'react';
+import { useAuth } from '../contexts/auth';
+import { usePlatform } from '../contexts/platform';
+import BookingListPreview from './BookingListPreview';
+import HomeModernIcon from './icons/HomeModernIcon';
+import { __ } from '../utils/helpers';
+
+const MyBookings = () => {
+  const { user } = useAuth();
+  const { platform } = usePlatform();
+  const myBookingsFilter = user && { where: { createdBy: user._id, status: ['pending', 'confirmed', 'checkedIn'] } };
+
+  const loadData = async () => {
+    await Promise.all([
+      platform.booking.get(myBookingsFilter)
+    ]);
+  }
+
+  useEffect(() => {
+    if (user && user._id){
+      loadData();
+    }
+  }, [user]);
+
+  const myBookings = platform.booking.find(myBookingsFilter);
+  const noBookings = myBookings && myBookings.count() === 0;
+  const error = myBookings && myBookings.get('error') // QUESTION: this Promise, how to get the error properly?
+
+  if(error) {
+    return <div className="validation-error">{ JSON.stringify(error) }</div>
+  }
+
+  if(!myBookings) {
+    return null;
+  }
+  
+  return (
+    <div className="columns">
+      <div className="col lg two-third">
+        <div className="page-header">
+          <h1 className="text-[32px] leading-[48px] font-normal border-b border-[#e1e1e1] border-solid pb-2 flex space-x-1 items-center">
```

Comment:
Let's not have hardcoded font sizes or colors please.

Reply:
Please, suggest the best way to set those, we might:
1. Hardcode them in css, inside a class, or generally `h1 {}` 
2. Get proportional values `px` > `rem` and put values in p1
3. Get the closest default `text-sm', `text-2xl`, etc. but the size will be different from the one set in Figma

Reply:
This should just inherit default website styles (so h1 {})